c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="\{\{mainCtrl.lang\}\}" ng-app="app" ng-controller="MainController as mainCtrl">
```

Comment:
the backslashes are necessary otherwise the scaffolding code complains that `mainCtrl.lang` is not defined when generating `ngeo.html` from `ngeo.html_tmpl`